src/features/import/hooks/useUIDataColumns.ts

```diff
@@ -238,6 +239,20 @@ export default function useUIDataColumns(
       }
     };
 
+    const assignTags = (mapping: TagColumn['mapping']) => {
```

Comment:
Learning question: TagColumn['mapping'], that ain't the same as TagColumn.mapping, is it?

Reply:
I _knew_ the hunch was there!